runtime/src/lib.rs

```diff
@@ -261,17 +261,17 @@ use sp_npos_elections::{ExtendedBalance};
 
 /// The type used to represent the kinds of proxying allowed.
 #[derive(
-	Copy,
-	Clone,
-	Eq,
-	PartialEq,
-	Ord,
-	PartialOrd,
-	Encode,
-	Decode,
-	RuntimeDebug,
-	MaxEncodedLen,
-	TypeInfo,
+Copy,
+Clone,
+Eq,
+PartialEq,
+Ord,
+PartialOrd,
+Encode,
+Decode,
+RuntimeDebug,
+MaxEncodedLen,
+TypeInfo,
```

Comment:
Is this change here by mistake? fix please.

Reply:
done